lib/teamsnap.rb

```diff
@@ -34,6 +35,15 @@ def init(opts = {})
       ##   create universally accessible TeamSnap.root_client
       self.root_client = TeamSnap::Client.new(:token => token)
 
+      ##   include any feature headers
+      if opts[:header]
```

Comment:
Should we support multiple headers? I know we only have one now but what if we want to support multiple?